### DISCUSSION TITLE
does support mysql & cpanel hosting?

### DISCUSSION THREAD

**Discussion by nabi009** [payloadcms/payload#1646] [General]
Hi, I am developing WordPress themes for many years and I am sick of it. I have searched on google and lastly, find the @payloadcms so awesome. as we are using Cpanel for hosting our sites and our clients do not effort to buy MongoDB separately for it also it looks so complex that  DB is on one server website is another server. is it possible to have the support of MySQL to host it in the same server and replace WordPress to @payload cms. thanks

**Comment by DanRibbens** [comment]
Right now it is Payload only works with MongoDB. We have a roadmap item for additional database support. You can follow along here: https://github.com/payloadcms/payload/discussions/287

It is understandable why you wouldn't want to move away from SQL, but Mongo does have it's advantages for content structures. One thing you could do is use MongoDB Atlas for the hosting of the database and still use your familiar tools for the web server. I know it is more to configure, but Atlas does have some options for multiple projects under one account and decent free options too.

This is up to you and your clients, but you should know there are some options. We also have Payload cloud coming, that will be the simplest way to host projects in the future.

**Reply by jmikrut** [reply]
Just a follow up note here, you can indeed put mongo on the same server as your application. We use DigitalOcean droplets a lot for this-one server for database and application.
